### PR TITLE
Fix impact icon selector

### PR DESCRIPTION
**Pull request by gannetson**

### CODE REVIEW

File: bluebottle/bluebottle_dashboard/static/jet/css/themes/goodup/_extra.scss [hunk 1]
```diff
@@ -91,3 +91,26 @@ span.help + p.help {
...
+}
+
+.icon-picker li img{
+  padding: 4px;
+  cursor: pointer;
+}
+
+
+.icon-picker li input:checked+img{
+  background-color: #64A03C66;
```
Comment (Pieter-Rees) [inline bluebottle/bluebottle_dashboard/static/jet/css/themes/goodup/_extra.scss:115]: Adding to @manuel-goodup feedback, this is no valid colorcode

Comment (gannetson) [reply bluebottle/bluebottle_dashboard/static/jet/css/themes/goodup/_extra.scss:115]: Why not? Because it has opacity?

Comment (manuel-goodup) [reply bluebottle/bluebottle_dashboard/static/jet/css/themes/goodup/_extra.scss]: @gannetson RGBA has more browser support than hex-with-opacity: 

https://caniuse.com/?search=rgba
https://caniuse.com/css-rrggbbaa

**Comment by gannetson** [comment]
Screenshot from 2022-08-30 11-14-40

File: bluebottle/bluebottle_dashboard/static/jet/css/themes/goodup/_extra.scss (hunk 1 above)
Comment (gannetson) [reply bluebottle/bluebottle_dashboard/static/jet/css/themes/goodup/_extra.scss]: There wasn't any design for this, no colors, an definitely any need for opacity.
I removed it ;-)

**Pull request merged**